### PR TITLE
[wip] get_by_id for view

### PR DESCRIPTION
**Pull request by t8y8**
Trying from the web editor

### CODE REVIEW

File: tableauserverclient/server/endpoint/views_endpoint.py [hunk 1]
```diff
@@ -36,6 +36,17 @@ def get(self, req_options=None, usage=False):
         all_view_items = ViewItem.from_response(server_response.content, self.parent_srv.namespace)
         return all_view_items, pagination_item
 
+    # Get 1 view by ID
+    @api(version="2.2")
```
Comment (shinchris) [inline tableauserverclient/server/endpoint/views_endpoint.py:40]: Querying view by ID was introduced with v3.1

**Comment by t8y8** [comment]
@shinchris Thanks! I think I'll have to remake it from the command line and force-push. The web editor automatically included the extra changes.

**Comment by shinchris** [comment]
@t8y8 sounds good, thanks!
It's a short notice, but we are planning on releasing a new version today, so it would be great to have this included if possible. If not, we can get it into the next release!

**Comment by bcantoni** [comment]
Recreated as a new PR #750

**Pull request closed without merging**